lldb/tools/lldb-dap/Transport.h

```diff
@@ -14,11 +14,11 @@
 #ifndef LLDB_TOOLS_LLDB_DAP_TRANSPORT_H
 #define LLDB_TOOLS_LLDB_DAP_TRANSPORT_H
 
+#include "DAPLog.h"
```

Comment:
is a forward declaration sufficient?

Reply:
Sure, added Log to DAPForward and swapped this include.